packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -139,6 +139,9 @@ pluginTester({
             ENABLE_FEATURE_NULL: null,
         },
     },
+    babelOptions: {
+        filename: __filename,
```

Comment:
What is this for?

Reply:
Great question! We need to pass the `filename` to babel because it is applying the `@babel/preset-typescript` preset when running these tests (even though we don't need that), and that preset is trying to ignore files that don't end with `ts` or `tsx`. Hardcoding this to `foo.ts` is probably less cryptic but you get the idea.

Reply:
We discussed yesterday that we may potentially only want to support running tests from the repo root, but I'm not sure we can do that until we figure out a way to disable presets and plugins at the package level. I'm not sure this is possible even if we enable local babelrc configurations because of the way babel merges those instead of overriding them. I've tried locally setting `preset` to `undefined`, `false`, and `[]`, without any luck.